pygmt/tests/test_grdfill.py

```diff
@@ -114,6 +114,31 @@ def test_grdfill_gridfill_dataarray(grid):
     npt.assert_array_equal(result[3:6, 3:5], bggrid[3:6, 3:5])
 
 
+def test_grdfill_hole(grid, expected_grid):
+    """
+    Test grdfill with different hole values.
```

Comment:
Hm. I am not sure if I understand what you mean with "different" here. As far as I know, there could be only one value used to define the nodes considered as holes at a time. Maybe "different" refers to the fact that the value for nodes considered as holes is set to a new value in this test?